web/overlay_manager.js

```diff
@@ -13,132 +13,139 @@
  * limitations under the License.
  */
 
-var OverlayManager = {
-  overlays: {},
-  active: null,
+const ESC_KEYCODE = 27;
```

Comment:
I'm not sure that adding this as a constant really brings much value; it's not like the keycode of `Esc` is going to change!?